packages/web/src/foundation/_reset.scss

```diff
@@ -38,3 +38,8 @@ table {
     border-collapse: collapse;
     border-spacing: 0;
 }
+
+a,
+a:visited {
```

Comment:
```suggestion
:any-link {
```

Reply:
Should have read this before trying `a:any-link` 🤦